### PR TITLE
Feature/#16 questions users endpoints

### PR DESCRIPTION
**Pull request by ricmtz**
-Questions users endpoints.
-./routes/index.js file remove
-Games questions and questions users routers added to ./index.js

### CODE REVIEW

File: routes/questions_users.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+// List all questions users.
+router.get('/questions_users', (req, res) => {
+    const questionsUsers = [
+        {
+            id: 123,
+            nickname: 'juan',
+            question: 'Que significa POO?',
+            options: ['Poder Oro y Osio', 'Puerta Ordenadas Operable', 'Programación Orienta a Objetos'],
+            corret: 'Programación Orienta a Objetos',
```
Comment (IrvingCJ) [inline routes/questions_users.js]: typo: "corret"

File: routes/questions_users.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    const json = {
+        response: 'Ok',
+        id: req.params.idQuestionsUsers,
+    };
+    res.status(200).send(json);
+});
+
+// Update question user.
+router.patch('/questions_users/:idQuestionsUsers', (req, res) => {
+    res.status(204).send('Data is updated');
```
Comment (IrvingCJ) [inline routes/questions_users.js]: "Data was updated" or "Data successfully updated"

File: routes/questions_users.js (hunk 1 above)
Comment (ricmtz) [reply routes/questions_users.js]: done

File: routes/questions_users.js (hunk 2 above)
Comment (ricmtz) [reply routes/questions_users.js]: done

**Pull request merged**